invisible_cities/cities/components.py

```diff
@@ -366,11 +368,10 @@ def mcsensors_from_file(paths     : List[str],
                                                        db_file    = db_file   ,
                                                        run_no     = run_number)
 
-        ## MC uses dummy timestamp for now
-        ## Only in case of evt splitting will be non zero
-        timestamp = 0
-
         for evt in mcinfo_io.get_event_numbers_in_file(file_name):
+
+            timestamp = create_timestamp(evt, rate)
+
```

Comment:
The timestamp that's in the files from data use milliseconds so we should probably be consistent.

Here we'd want to do `timestamp = creat_timestamp(evt, rate) / units.ms`

This solution relies on the correct use of the units in the configuration of the job (I've put a comment to add units to the configuration)

Reply:
Since I'm not used to it, I didn't even think about adding units. I'll keep it in mind next times.